src/HomeWork.java

```diff
@@ -0,0 +1,91 @@
+
+public class HomeWork {
+    public static int calculateAbsoluteDifference(int a, int b) {
+        return Math.abs(a - b);
+    }
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+        int result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+        a = -8;
+        b = 7;
+        result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+    }
+}
+
+class FirstChar {
+    public static char getFirstCharacter(String inputString) {
```

Comment:
Read the description of task carefully. Only first method was supposed to be static.